.github/workflows/package.yml

```
@@ -16,17 +16,17 @@ jobs:
     steps:
       
     - name: Checkout code
-      uses: actions/checkout@v3
+      uses: actions/checkout@v4
         
 
     - name: Set up Java
-      uses: actions/setup-java@v3
+      uses: actions/setup-java@v4
       with:
           java-version: '17'
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)